packages/manager/modules/office/src/dashboard/microsoft-office-license-postpaid.service.js

```diff
@@ -0,0 +1,71 @@
+import { POSTPAID_BASE_URL } from './microsoft-office-license.constants';
+
+export default class MicrosoftOfficeLicensePostpaidService {
+  /* @ngInject */
+  constructor($http, OvhHttp) {
+    this.$http = $http;
+    this.ovhHttp = OvhHttp;
+  }
+
+  get(serviceName) {
+    return this.$http.get(`${POSTPAID_BASE_URL}/${serviceName}`);
```

Comment:
i think it's better to have the right response data directly.
```suggestion
    return this.$http.get(`${POSTPAID_BASE_URL}/${serviceName}`)
                              .then(({data}) => data);
```

Reply:
The API calls are split into `prepaid` and `postpaid` services since there are 2 different sets of endpoints. But in future, `postpaid` end point will be merged into `prepaid`. So the implementation is done keeping in my mind to minimise the amount of work during  `postpaid` service removal ( We can merge `prepaid` service and `microsoft-office-license.service`)